src/app/pages/campaigns/tabs/npcs/npcs.component.html

```diff
@@ -18,6 +18,9 @@ <h2 class="rh24 bmr-40">NPCs</h2>
         <div class="flex-col flex-dynamic">
           <input type="text" class="form-control" (ngModelChange)="self.touch()" [(ngModel)]="npc.name" placeholder="Name">
         </div>
+        <div class="flex-col flex-static">
+          <button title="Clone NPC" class="g-icon" (click)="self.methods.cloneNPC(npc, 'npc')"><fa-icon [icon]="['fas', 'copy']"></fa-icon></button>
```

Comment:
I don't think we should make it so cloned NPCs are called `'npc'`. Maybe the NPC's name but preceded by `'Copy of '`?

Reply:
Sure thing! Hey, what about not passing this 'npc' type at all and checking (on campaign service) if the object is an instance of CampaignFoe instead? Would that work?